common/rabbitmq/templates/deployment.yaml

```diff
@@ -133,4 +138,9 @@ spec:
         - name: container-init
           configMap:
             name: {{ template "fullname" . }}-bin
+      {{- if $.Values.customConfig }}
```

Comment:
We don't seem to use `$.Values` but only `.Values` in the other parts of the file - any reason to change that here in the `if`?

Reply:
you are right actually in this context is doesn't make any difference to use .`Value` or` $.Value`
changed it to `.Value` to keep consistent